FluentAvalonia/UI/Controls/ColorPickerButton/ColorPickerButton.properties.cs

```diff
@@ -200,7 +202,24 @@ public bool ShowAcceptDismissButtons
 			set => SetValue(ShowAcceptDismissButtonsProperty, value);
 		}
 
-		private bool _isCompact = true;
+        /// <summary>
+        /// Raised when the color change was confirmed and the flyout closes.
+        /// </summary>
+        public event TypedEventHandler<ColorPickerButton, Color> FlyoutConfirmed;
+        /// <summary>
+        /// Raised when the color change was dismissed and the flyout closes.
+        /// </summary>
+        public event TypedEventHandler<ColorPickerButton, object> FlyoutDismissed;
```

Comment:
Change `object` to `EventArgs` (here and below). For `FlyoutConfirmed` above, probably should use `ColorChangedEventArgs`

I'm assuming you copied these from (or at least based them on) PickerFlyoutPresenter events which I must've missed when I made this change everywhere else, so that's my fault for the inconsistency

Reply:
I grabbed them from the `PickerFlyoutPresenter` yep
Made sure everything is using EventArgs now (which makes more sense, I should've noticed that myself)